README.md

```diff
@@ -90,6 +90,73 @@ $decrypted = $rsa->decrypt($encrypted);
 var_dump($decrypted); // 'abc123'
 ```
 
+Don't have key files already? No problem - you can simply create new ones :
+
+```php
+$rsa = new Pikirasa\RSA('path/to/nonexistent_public.pem', 'path/to/nonexistent_private.pem');
+$rsa->create();  // creates new keys in the new key files
+
+$data = 'abc123';
+$encrypted = $rsa->encrypt($data);
+$decrypted = $rsa->decrypt($encrypted);
+var_dump($decrypted); // 'abc123'
+```
+
+Need a key size other than the default of 2048 bits? Simply pass the size you need as the first parameter of `$rsa->create()`.
+
+Pikirasa won't overwrite existing key files unless you pass `true` to the second parameter of `$rsa->create()`.
+
+If you prefer to work with key strings over key files, you can create keys that way, too :
+
+```php
+$rsa = new Pikirasa\RSA(null, null);
+$rsa->create();  // creates new keys as strings
+
+$data = 'abc123';
+$encrypted = $rsa->encrypt($data);
+$decrypted = $rsa->decrypt($encrypted);
+var_dump($decrypted);
+          // 'abc123'
+var_dump($publicKey);
+          // -----BEGIN PUBLIC KEY-----
+          // MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7o9A47JuO3wgZ/lbOIOs
+          // Xc6cVSiCMsrglvORM/54StFRvcrxMi7OjXD6FX5fQpUOQYZfIOFZZMs6kmNXk8xO
+          // hgTmdMJcBWolQ85acfAdWpTpCW29YMvXNARUDb8uJKAApsISnttyCnbvp7zYMdQm
+          // HiTG/+bYaegSXzV3YN+Ej+ZcocubUpLp8Rpzz+xmXep3BrjBycAE9z2IrrV2rlwg
+          // TTxU/B8xmvMsToBQpAbe+Cv130tEHsyW4UL9KZY1M9R+UHFPPmORjBKxSZvjJ1mS
+          // UbUYN6PmMry35wCaFCfQoyTDUxBfxTGYqjaveQv4sxx0uvoiLXHt9cAm5Q8KJ+8d
+          // FwIDAQAB
+          // -----END PUBLIC KEY-----
+var_dump($privateKey);
```

Comment:
Some here with this variable. Where is it declared?

Reply:
Same as above.